com.avaloq.tools.ddk.xtext.ui/src/com/avaloq/tools/ddk/xtext/ui/validation/preferences/ValidPreferenceConstants.java

```diff
@@ -10,37 +10,17 @@
  *******************************************************************************/
 package com.avaloq.tools.ddk.xtext.ui.validation.preferences;
 
-import org.eclipse.jface.resource.ImageDescriptor;
-
-
 /**
  * Constants for the ValidPreference and associated classes.
  */
 public final class ValidPreferenceConstants {

```

Comment:
Why is this class still needed?

Reply:
Transferred the PREFERENCE_SEPARATOR to ValidValidatorFragment. Removed the class.